app/views/locker_applications/_form.html.erb

```diff
@@ -16,37 +16,35 @@
     </card>
   <% end %>
   <card size="full-width">
-   <grid-container>
-   <grid-item columns="lg-3 md-6 sm-12">
+   <grid-container class="locker-form">
+   <grid-item columns="lg-4 md-6 sm-12">
      <input-select label="Preferred Size" name="locker_application[preferred_size]"
                       id="locker_application_preferred_size"
                       width="expand"
                       value="<%= locker_application.preferred_size %>"
                       :options="<%= locker_application.size_choices.to_json %>"
         ></input-select>
    </grid-item>
-   <grid-item columns="lg-3 md-6 sm-12">
-     <input-select label="Preferred General Area" name="locker_application[preferred_general_area]"
+   <grid-item columns="lg-4 md-6 sm-12">
+     <input-select label="Preferred Floor" name="locker_application[preferred_general_area]"
                       id="locker_application_preferred_general_area"
                       width="expand"
                       value="<%= locker_application.preferred_general_area %>"
                       :options="<%= locker_application.floor_choices.to_json %>"
         ></input-select>
    </grid-item>
-   <grid-item columns="lg-3 md-6 sm-12">
-
-    <%= check_box :locker_application, :accessible, {}, "true", "false" %>
-    <%= label(:locker_application, :accessible, "Accessible") %>
-  
-   </grid-item>
-   <grid-item columns="lg-3 md-6 sm-12">
+   <grid-item columns="lg-4 md-6 sm-12">
      <input-select label="Semester of Occupancy" name="locker_application[semester]"
                       id="locker_semester"
                       width="expand"
                       value="<%= locker_application.semester %>"
                       :options="<%= locker_application.semester_choices.to_json %>"
         ></input-select>
-
+   </grid-item>
+   <grid-item columns="lg-12 md-12 sm-12">
+     <%= check_box :locker_application, :accessible, {}, "true", "false" %>
+     <%= label(:locker_application, :accessible, "Accessible") %>
+     <div>For example, near an elevator or keyed entry as opposed to a combination lock</div>
```

Comment:
@rladdusaw during the checkin we said to move the 'Accessible' on top, as the first field. Unless someone else remembers differently.